script/measure_vram.sh

```diff
@@ -11,8 +11,11 @@ run_command () {
     done
 }
 
-run_command "baffo32/decapoda-research-llama-7B-hf" "llama-1-7b" "--fix_decapoda_config"
-run_command "nota-ai/st-llama-1-5.5b-ppl" "st-llama-1-5.5b-ppl" "--fix_decapoda_config"
+#run_command "baffo32/decapoda-research-llama-7B-hf" "llama-1-7b" "--fix_decapoda_config"
+#run_command "nota-ai/st-llama-1-5.5b-ppl" "st-llama-1-5.5b-ppl" "--fix_decapoda_config"
 
-run_command "lmsys/vicuna-13b-v1.3" "vicuna-13b-v1.3" ""
-run_command "nota-ai/st-vicuna-v1.3-10.5b-ppl" "st-vicuna-v1.3-10.5b-ppl" ""
\ No newline at end of file
+#run_command "lmsys/vicuna-13b-v1.3" "vicuna-13b-v1.3" ""
+#run_command "nota-ai/st-vicuna-v1.3-10.5b-ppl" "st-vicuna-v1.3-10.5b-ppl" ""
+
+TOKENIZER="nota-ai/only-prune_st-vicuna-v1.3-3.7b-ppl"
+run_command "quantized_models/GPTQ/st-vicuna-v1.3-3.7b-ppl-CPT-iter-2208000" "st-vicuna-v1.3-3.7b-ppl-CPT-iter-2208000-GPTQ" "--tokenizer $TOKENIZER"
```

Comment:
Please change this line to use the results of `script/quantize_gptq_vicuna-7b.sh`. The directory name seems inadequate.

Reply:
Question: Is specifying the tokenizer necessary? `src/quantize_gptq.py` seems to save the tokenizer in `quantized_model_dir` at [this line](https://github.com/Nota-NetsPresso/shortened-llm/blob/ee80366346383a2686ec153a474bf7951547740e/src/quantize_gptq.py#L98)

Reply:
As you pointed out, we don't need to specify a tokenizer since `src/quantize_gptq.py` save tokenizer when saving quantized model.